src/lib/is-browser.ts

```diff
@@ -1,5 +1,17 @@
-const isStandardBrowserEnv = () =>
-	typeof window !== 'undefined' && typeof window.document !== 'undefined'
+const isStandardBrowserEnv = () => {
+	if (typeof window !== 'undefined') {
+		if (
+			navigator.userAgent.toLowerCase().indexOf(' electron/') > -1 &&
+			Object.prototype.hasOwnProperty.call(process.versions, 'electron')
```

Comment:
I think there should be an `||` here and not an `&&` ?

Reply:
Also I'm not sure it is correct to return false in case `navigator.userAgent.toLowerCase().indexOf(' electron/') > -1`.

Based on https://github.com/electron/electron/issues/2288#issuecomment-611231970 seems that means we are in the browser env (rendered script), but `process.versions.hasOwnProperty('electron')` means we are in nodejs env (main script)

Reply:
@axi92 I would like to know here how to detect when context isolation is true, maybe it was correct to keep the `&&` now that I think about this because seems this happens only when there is no context isolation and so the isBrowser returns true because window is defined... I think reverting back to `&&` is the correct way. Please add a comment that explains this above the code you added

Reply:
When context isolation is true electron cant use require in the render.js so there is no way to load the mqtt lib, or am I wrong?
I checked different combinations for the `webPreferences` but only one configuration is working:
```
        webPreferences: {
            sandbox: false,
            nodeIntegration: true,
            contextIsolation: false, 
        },
```
If this config is not set in that way I can't use `require()` in the render.js.
In main.js it is always working because that is always node context.